apis/broker/ts/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.0.1
\ No newline at end of file
+5.1.1
```

Comment:
I did an investigation on using a generic IPFS API (instead of the specific Pinata API) with the hope it would be able to connect to a mocked IPFS node.

Unfortunaltey, I misunderstood the whole purpose of the mocked node and the IPFS API - it wasn't for uploading, it was to mock pinning (expecting that a local IPFS node would have already uploaded the actual data)

But since it's here, we might as well leave hte upgrade in because it has fixes for the version of TS we are using